application.py

```diff
@@ -1,24 +1,31 @@
-import argparse
-import os
-
-from app import application
-
-# Only enable Flask debugging if an env var is set to true
-application.debug = os.environ.get('FLASK_DEBUG') in ['true', 'True']
-
-# Get application version from env
-app_version = os.environ.get('APP_VERSION')
-
-# Get cool new feature flag from env
-enable_cool_new_feature = os.environ.get('ENABLE_COOL_NEW_FEATURE') in ['true', 'True']
-
+"""
+Application Entry
+====================
+"""
 
 if __name__ == '__main__':
+    import argparse
+
     parser = argparse.ArgumentParser(description='arguments options:')
     parser.add_argument(
         '-p', '--port', type=int, default=5000, help='Specify listening port.'
     )
     args = parser.parse_args()
     port = args.port
 
-    application.run(host='0.0.0.0', port=port)
+    import logging.config
+    import yaml
+    import os
+
+    if not os.path.exists('./logs'):
+        os.makedirs('./logs')
+
+    with open('logging-conf.yaml', 'r') as f:
+        config = yaml.safe_load(f.read())
+        logging.config.dictConfig(config)
```

Comment:
I already see an `import logging.config` in this file.  Should we just use config directly?

Reply:
I am not sure what you mean. `import logging.config` is just a module import.

Reply:
I'm not certain on python syntax. If that's the case, should we change the name? Say you created a variable called `config`, and import a module with the same name (logging.confg).

Reply:
don't worry, python will handle it correctly.